Src/DependencyCollector/Shared/Implementation/ProfilerSqlProcessingBase.cs

```diff
@@ -101,6 +119,24 @@ public object OnEndForTwoParameters(object context, object returnValue, object t
         }
 
         /// <summary>
+        /// On end async callback for methods with 2 parameter.
+        /// </summary>
+        public object OnEndAsyncForTwoParameters(object context, object returnValue, object thisObj, object parameter1)
+        {
+            this.OnEndAsync(returnValue, thisObj);
+            return returnValue;
+        }
+
+        /// <summary>
+        /// On end async callback for methods with 2 parameter.
```

Comment:
no, that's OnEnd that sends event only in case of exception.

---
In reply to: [98065128](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/298#discussion_r98065128) [](ancestors = 98065128)